appyx-components/modal/common/src/commonMain/kotlin/com/bumble/appyx/components/modal/operation/Show.kt

```diff
@@ -0,0 +1,39 @@
+package com.bumble.appyx.components.modal.operation
+
+import androidx.compose.animation.core.AnimationSpec
+import com.bumble.appyx.components.modal.Modal
+import com.bumble.appyx.components.modal.ModalModel
+import com.bumble.appyx.interactions.Parcelize
+import com.bumble.appyx.interactions.core.model.transition.BaseOperation
+import com.bumble.appyx.interactions.core.model.transition.Operation
+
+@Parcelize
+class Show<InteractionTarget : Any>(
+    override var mode: Operation.Mode = Operation.Mode.KEYFRAME
+) : BaseOperation<ModalModel.State<InteractionTarget>>() {
+
+    override fun isApplicable(state: ModalModel.State<InteractionTarget>) =
+        state.created.isNotEmpty()
+
+    override fun createTargetState(fromState: ModalModel.State<InteractionTarget>): ModalModel.State<InteractionTarget> =
+        fromState.copy(
+            created = fromState.created.dropLast(1),
+            modal = fromState.created.last(),
+            fullScreen = null,
+            destroyed = fromState.destroyed + listOfNotNull(fromState.modal, fromState.fullScreen)
+        )
+
+    override fun createFromState(baseLineState: ModalModel.State<InteractionTarget>) = baseLineState
+}
+
+fun <InteractionTarget : Any> Modal<InteractionTarget>.show(
+    animationSpec: AnimationSpec<Float> = defaultAnimationSpec,
+    mode: Operation.Mode = Operation.Mode.KEYFRAME
```

Comment:
Default mode for all of these operations could be `IMMEDIATE`.

Reply:
Updated for all operations